gateway/api/views.py

```diff
@@ -255,6 +268,31 @@ def run(self, request):
             job_serializer = self.get_serializer_job_class()(job)
         return Response(job_serializer.data)
 
+    @action(methods=["POST"], detail=False)
+    def set_public(self, request):
```

Comment:
I think django out of the box is giving update API on every speicified field in serializer🤔 

Can we check that? Not to create redundant enpoinds.